kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -78,7 +78,8 @@ def unlocalized_url(viewname, *args, **kwargs):
     """Helper for Django's ``reverse`` in templates.
 
     Uses django's default reverse."""
-    return django_reverse(viewname, args=args, kwargs=kwargs)
```

Comment:
It looks like this function is only used for canonical URLs, and so now it seems poorly named. How about we go ahead and change it to something like `get_canonical_url()` or `canonicalize()`. Looks like it could just take the URL as an argument too, which would mean usage like `{{ canoncialize(document.get_absolute_url() }}`.